diff --git a/source/service_multinet/Puma6_plat/handle_sw.c b/source/service_multinet/Puma6_plat/handle_sw.c
index f67ae3fc0..3cd086edd 100755
--- a/source/service_multinet/Puma6_plat/handle_sw.c
+++ b/source/service_multinet/Puma6_plat/handle_sw.c
@@ -40,6 +40,8 @@
 int hdl_sw_sysevent_fd;
 token_t hdl_sw_sysevent_token;
 
+INT swctl(const int command, const int port, const int vid, const int membertag, const int vlanmode, const int efm, const char *mac, const char *magic){return 0;}
+
 void check_for_dependent_ports(char *port, int *tag, int *atom_port, int *ext_port)
 {
 	if(!strncmp(port, "ath", 3) || (!strncmp(port, "sw_6", 4)))
